src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt

```diff
@@ -88,7 +93,9 @@ MembersMustExist : Member 'System.Data.SqlClient.SqlParameter.UdtTypeName.get()'
 MembersMustExist : Member 'System.Data.SqlClient.SqlParameter.UdtTypeName.set(System.String)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Data.SqlClient.SqlParameterCollection.Add(System.String, System.Data.SqlDbType, System.Int32, System.String)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Data.SqlClient.SqlParameterCollection.Add(System.String, System.Object)' does not exist in the implementation but it does exist in the contract.
+Compat issues with assembly System.Drawing:
+CannotRemoveBaseTypeOrInterface : Type 'System.Drawing.Printing.PrintingPermissionAttribute' does not implement interface 'System.Runtime.InteropServices._Attribute' in the implementation but it does in the contract.
 Compat issues with assembly System.Xml:
 MembersMustExist : Member 'System.Xml.Xsl.XslCompiledTransform.TemporaryFiles.get()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Xml.Xsl.XslTransform.Load(System.Xml.XPath.XPathNavigator, System.Xml.XmlResolver, System.Security.Policy.Evidence)' does not exist in the implementation but it does exist in the contract.
-Total Issues: 82
+Total Issues: 93
```

Comment:
I think I've royally goofed up my master at this point. Basically I think I left pull.rebase=true which is my normal mode of operation. When syncing my fork with the upstream I rebased instead of merged so now git believes there are different histories and it causes all kinds of conflicts or refuses to merge. This branch seems Ok for now, but as I noticed with the .csproj file above the diffs between my local branch and master aren't working correctly.

I tried refloging and resetting all the way back but still didn't have any luck. 

TLDR; I think I need to delete my fork and reconfigure it (properly).  Alternatively we could try to merge this PR and then I can fix things from there.